src/App.vue

```diff
@@ -1,5 +1,16 @@
 <template>
-  <div>
+  <main>
+    <div class="slang-words__wrapper">
+      <Words
+        header="Militære forkortelser"
+        subheader="Noe"
+        inputPlaceholder="Noe"
+        table1Header="Noe"
+        table2Header="Noe"
+        :paginationIndex="30"
```

Comment:
binds med : må ligge over vanlige attributter
Hierakiet vi bruker er:
refs
if else
for
:binds
attributter
@events